### PR TITLE
Adding additional tests for different data source types

### PR DESCRIPTION
**Pull request by MayaKirova**
Adding additional tests for different data source types

### CODE REVIEW

File: tests/unit/dataSource/properties/tests.html [hunk 1]
```diff
@@ -126,6 +126,62 @@
...
+			equal(jsonpDs.type(), "json", "Check if dataSource type is correct.");
+			equal(jsonpDs.dataView().length, 2, "DataView should contain 2 records.");			
+			
+			jsonpDs = new $.ig.JSONDataSource();
+
+			equal(jsonpDs.type(), "json", "Check if dataSource type is correct.");			
+		});
+		
+		test("Test creating HtmlTableDataSource ", function () {
+			var htmltableDs = new $.ig.HtmlTableDataSource ();
```
Comment (mpavlinov) [inline tests/unit/dataSource/properties/tests.html]: Feed it with some HTML and see if it gets parsed correctly.

File: tests/unit/dataSource/properties/tests.html [hunk 2]
```diff
@@ -126,6 +126,62 @@
...
+			equal(htmltableDs.type(), "htmlTableDom", "Check if dataSource type is correct.");			
+		});
+		
+		test("Test creating ArrayDataSource  ", function () {
+			var arrayDs = new $.ig.ArrayDataSource();
```
Comment (mpavlinov) [inline tests/unit/dataSource/properties/tests.html]: Feed it with an array data and see if it is available in the dataView()

File: tests/unit/dataSource/properties/tests.html [hunk 3]
```diff
@@ -126,6 +126,62 @@
...
+			equal(arrayDs.type(), "array", "Check if dataSource type is correct.");			
+		});
+		
+		test("Test creating RemoteDataSource   ", function () {
+			var arrayDs = new $.ig.RemoteDataSource ();
```
Comment (mpavlinov) [inline tests/unit/dataSource/properties/tests.html]: Same here. Give it an Url.

**Comment by MayaKirova** [comment]
There are already similar tests under dataSource/schemas so there's no need to verify this again.

**Pull request merged**